lib/worker_base.js

```diff
@@ -24,6 +25,51 @@ class WorkerBase {
   init() {
     throw new Error("'init' method need to be overriden")
   }
+
+  /**
+   * Should be overriden depending on blockchain implementation.
+   *
+   * @returns Number of new requests made towards the Node endpoint. Used for metrics purposes.
+   */
+  getNewRequestsCount() {
+    return 1
+  }
```

Comment:
Shouldn't this be 0 in the base class?

Reply:
The presumption is that on each 'work loop' the Full Node would be queried at least once, that's true for all blockchains. I think 1 is less wrong that 0, but I guess I can throw exception in the base class and force overwrite. What do you think?